hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -131,6 +131,32 @@ <h2 class="formH2">Your application has been submitted!</h2>
                 {% endif %}
             </div>
 
+            {% if application is not none %}
+            <div class="borderTopDiv z-depth-3">
```

Comment:
make the theme consistent with the FAQ section. (i.e bold the question, and make the answer below it)
<img width="895" alt="CleanShot 2022-12-20 at 23 51 51@2x" src="https://user-images.githubusercontent.com/36619969/208824343-b0609139-adfd-497e-9b50-6b55ed53f1a4.png">